GVFS/GVFS.FunctionalTests/Tests/MultiEnlistmentTests/ConfigVerbTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using GVFS.FunctionalTests.Tools;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace GVFS.FunctionalTests.Tests.MultiEnlistmentTests
+{
+    [TestFixture]
+    [NonParallelizable]
+    [Category(Categories.MacTODO.M4)]
+    public class ConfigVerbTests : TestsWithMultiEnlistment
+    {
+        [OneTimeSetUp]
+        public void DeleteAllSettings()
+        {
+            string configFilePath = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData),
+                "GVFS",
+                "gvfs.config");
+
+            if (File.Exists(configFilePath))
+            {
+                File.Delete(configFilePath);

```

Comment:
how about adding a prefix string to all configs used for CRUD from inside the Functional Test rather than delete and create a fresh config file. The FT would not touch the regular pre-existing config settings. Updated the PR with this option - using "FunctionTest_" prefix for the FT settings.